packages/contracts/test/StabilityPoolTest.js

```diff
@@ -2448,321 +1875,26 @@ contract('StabilityPool', async accounts => {
-      for (const [frontEnd, G] of Object.entries(frontEndToG)) {
-        const snapshot = await stabilityPool.frontEndSnapshots(frontEnd)
-
-        // Check snapshots are the expected values
-        assert.equal(snapshot[0], '0')  // S (should always be 0 for front ends)
-        assert.isTrue(snapshot[1].eq(P_Before))  // P 
-        assert.isTrue(snapshot[2].eq(G))  // G
-        assert.equal(snapshot[3], '0')  // scale
-        assert.equal(snapshot[4], '0')  // epoch
-      }
-    })
-
-    it("withdrawFromSP(), full withdrawal: removes deposit's front end tag", async () => {
-      await openTrove({ extraLUSDAmount: toBN(dec(100000, 18)), ICR: toBN(dec(10, 18)), extraParams: { from: whale } })
-
-      // Whale transfers to A, B 
-      await lusdToken.transfer(A, dec(10000, 18), { from: whale })
-      await lusdToken.transfer(B, dec(20000, 18), { from: whale })
-
-      //C, D open troves
-      await openTrove({ extraLUSDAmount: toBN(dec(30000, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: C } })
-      await openTrove({ extraLUSDAmount: toBN(dec(40000, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: D } })
-
-      // A, B, C, D make their initial deposits
-      await stabilityPool.provideToSP(dec(10000, 18), frontEnd_1, { from: A })
-      await stabilityPool.provideToSP(dec(20000, 18), ZERO_ADDRESS, { from: B })
-      await stabilityPool.provideToSP(dec(30000, 18), frontEnd_2, { from: C })
-      await stabilityPool.provideToSP(dec(40000, 18), ZERO_ADDRESS, { from: D })
-
-      // Check deposits are tagged with correct front end 
-      const A_tagBefore = await getFrontEndTag(stabilityPool, A)
-      const B_tagBefore = await getFrontEndTag(stabilityPool, B)
-      const C_tagBefore = await getFrontEndTag(stabilityPool, C)
-      const D_tagBefore = await getFrontEndTag(stabilityPool, D)
-
-      assert.equal(A_tagBefore, frontEnd_1)
-      assert.equal(B_tagBefore, ZERO_ADDRESS)
-      assert.equal(C_tagBefore, frontEnd_2)
-      assert.equal(D_tagBefore, ZERO_ADDRESS)
-
-      // All depositors make full withdrawal
-      await stabilityPool.withdrawFromSP(dec(10000, 18), { from: A })
-      await stabilityPool.withdrawFromSP(dec(20000, 18), { from: B })
-      await stabilityPool.withdrawFromSP(dec(30000, 18), { from: C })
-      await stabilityPool.withdrawFromSP(dec(40000, 18), { from: D })
-
-      // Check all deposits now have no front end tag
-      const A_tagAfter = await getFrontEndTag(stabilityPool, A)
-      const B_tagAfter = await getFrontEndTag(stabilityPool, B)
-      const C_tagAfter = await getFrontEndTag(stabilityPool, C)
-      const D_tagAfter = await getFrontEndTag(stabilityPool, D)
-
-      assert.equal(A_tagAfter, ZERO_ADDRESS)
-      assert.equal(B_tagAfter, ZERO_ADDRESS)
-      assert.equal(C_tagAfter, ZERO_ADDRESS)
-      assert.equal(D_tagAfter, ZERO_ADDRESS)
-    })
-
-    it("withdrawFromSP(), full withdrawal: zero's depositor's snapshots", async () => {
```

Comment:
do we still want this test? (not sure if snapshot can refer to both front and and depositor, seems like yes)

Reply:
the test is still there it's just up at line 1878